tests/data/test_inputs.py

```diff
@@ -65,4 +65,4 @@ def test_too_many_points_simplify():
 	geometry = gpd.GeoSeries(polygon)
 
 	checked = check_and_format_shape(geometry, simplify=True)
-	assert checked is not None
+	checked.contains(geometry)
```

Comment:
assert True?

Reply:
I think it's like 
```python
assert checked.contains(geometry).all()
```

Reply:
doh, that's what I get for rushing. I also had to make a more meaningful geometry, as the check you suggested didn't work initially as I just made a straight line :)